sos/report/plugins/__init__.py

```diff
@@ -3004,24 +3004,9 @@ def _get_scls(self):
         return [scl.strip() for scl in output.splitlines()]
 
     def convert_cmd_scl(self, scl, cmd):
-        """wrapping command in "scl enable" call and adds proper PATH
+        """wrapping command in "scl enable" call
         """
-        # load default SCL prefix to PATH
-        prefix = self.policy.get_default_scl_prefix()
-        # read prefix from /etc/scl/prefixes/${scl} and strip trailing '\n'
-        try:
-            prefix = open('/etc/scl/prefixes/%s' % scl, 'r').read()\
-                     .rstrip('\n')
-        except Exception as e:
-            self._log_error("Failed to find prefix for SCL %s using %s: %s"
-                            % (scl, prefix, e))
-
-        # expand PATH by equivalent prefixes under the SCL tree
-        path = os.environ["PATH"]
-        for p in path.split(':'):
-            path = '%s/%s%s:%s' % (prefix, scl, p, path)
-
-        scl_cmd = "scl enable %s \"PATH=%s %s\"" % (scl, path, cmd)
+        scl_cmd = "scl enable %s \"%s\"" % (scl, cmd)
```

Comment:
Is `scl` doing its own PATH handling something it has always done, or is this a somewhat new development? We've had SCL collections in the project for years (even though the number of those collections is fairly low), without a report of an issue so this makes me wonder about other `scl` collections outside of the one that triggered this specific change.